src/early_multicellular_stage/editor/EarlyMulticellularEditor.cs

```diff
@@ -435,14 +435,14 @@ private void OnStartEditingCellType(string name, bool switchTab)
             return;
         }
 
-        var newTypeToEdit = EditedSpecies.CellTypes.First(c => c.TypeName == name);
+        var newTypeToEdit = EditedSpecies.CellTypes.FirstOrDefault(c => c.TypeName == name);
 
         // Only reinitialize the editor when required
         if (selectedCellTypeToEdit == null || selectedCellTypeToEdit != newTypeToEdit)
         {
             selectedCellTypeToEdit = newTypeToEdit;
 
-            GD.Print("Start editing cell type: ", selectedCellTypeToEdit.TypeName);
+            GD.Print("Start editing cell type: ", selectedCellTypeToEdit?.TypeName);
```

Comment:
Also this message should be vastly different if there's no type selected. A message saying "Start editing cell type: " is going to be really confusing to realize that literally nothing there means that previous cell type to be edited was actually cleared, rather than new edit just began.